### PR TITLE
standardize script commands to small letters

### PR DESCRIPTION
**Pull request by AnnieRuru**
[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style
### Pull Request Prelude
- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR will be closed if the above-mentioned criteria are not fulfilled.

### Issues addressed
http://herc.ws/board/topic/16526-some-upcoming-hercules-features/?do=findComment&comment=90516

### Changes Proposed
All script commands should be in small letters

### Affected Branches
* Master

### Known Issues and TODO List
I don't want to memorize another special capital letter case
"All" and "all" is already enough for us -> https://github.com/HerculesWS/Hercules/issues/784
--> standardize into "all" for script commands -> https://github.com/HerculesWS/Hercules/pull/2380

~so if Haru say again this isn't an issue, feel free to close this PR~
for standardization, script commands has to be small letters

### CODE REVIEW

**Comment by HerculesWSAPI** [comment]
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/2374)

**Comment by dastgirp** [comment]
Thanks, I feel haru too wanted script commands in small case.

About "all" that should be case insensitive for now and prefer to have it recommendation that it should be small case

**Comment by AnnieRuru** [comment]
on 2nd thought, actually we can just add "all" and "All" in the script.c
```
 src/map/script.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/map/script.c b/src/map/script.c
index 7578fcdcc..b1e0a5135 100644
--- a/src/map/script.c
+++ b/src/map/script.c
@@ -11031,7 +11031,7 @@ static BUILDIN(killmonster)
 	int16 m,allflag=0;
 	mapname=script_getstr(st,2);
 	event=script_getstr(st,3);
-	if(strcmp(event,"All")==0)
+	if (strcmp(event, "All") == 0 || strcmp(event, "all") == 0)
 		allflag = 1;
 	else
 		script->check_event(st, event);
@@ -13991,7 +13991,7 @@ static BUILDIN(mobcount)
 	mapname=script_getstr(st,2);
 	event=script_getstr(st,3);
 
-	if( strcmp(event, "all") == 0 )
+	if (strcmp(event, "all") == 0 || strcmp(event, "All") == 0)
 		event = NULL;
 	else
 		script->check_event(st, event);
```
although after so many years we have already adapted to this ... but its better be late than never

**Comment by dastgirp** [comment]
Or change to strcmpi

**Comment by 4144** [comment]
strcmpi will allow ``all``, ``All``, ``aLL`` and other combinations...

File: src/map/script.c [hunk 1]
```diff
@@ -25792,10 +25792,10 @@ static void script_parse_builtin(void)
 
 		BUILDIN_DEF(itempreview, "i"),
 		BUILDIN_DEF(enchantitem, "iii"),
-		BUILDIN_DEF(expandInventoryAck, "i?"),
```
Comment (4144) [inline src/map/script.c]: Probably wrong names better add with BUILDIN_DEF_DEPRECATED?

Comment (AnnieRuru) [reply src/map/script.c]: huh ???
you are the one that made the commit ...
https://github.com/HerculesWS/Hercules/blob/master/npc/other/inventory_expansion.txt
and it used in this script, why deprecate it ?

Comment (dastgirp) [reply src/map/script.c]: @AnnieRuru , 4144 probably meant that deprecate commands with wrong names (camel case words) and add new lines for correct name.

Comment (AnnieRuru) [reply src/map/script.c]: erm ... this script command was just implemented recently ... right ?
its not like this script command already wide-spread ...

Comment (4144) [reply src/map/script.c]: near 2 months i think. merged to hercules 1 month ago

Comment (AnnieRuru) [reply src/map/script.c]: hmm ... some in-game testing
```
-	script	khfsdkhjsf	FAKE_NPC,{
OnPCLoginEvent:
	debugmes getInventorySize +"";
}
```
already throw error like
```
[Error]: script_add_str: detected possible use of wrong case in a script. Found
'getInventorySize', probably meant to be 'getinventorysize' (in 'npc/zzz.txt').
```
you sure still need to fix this ?

Comment (AnnieRuru) [reply src/map/script.c]: 1 month++ bump

I have tested this PR will throw error on **parse time**, with the error shown in above post
@4144

Comment (4144) [reply src/map/script.c]: please decrecated all commands with wrong case.
for example: ``BUILDIN_DEF2_DEPRECATED(expandinventoryack, "expandInventoryAck", "i?")``

Comment (AnnieRuru) [reply src/map/script.c]: nonono, please test this PR, this doesn't need to deprecate
there is already the wrong case detection in the script engine to throw that error already

also, when I tested the `BUILDIN_DEF2_DEPRECATED` like you said, when I did it before
the small letter case got deprecated as well for some reason ...

try it yourself

**Pull request merged**